server/service.go

```diff
@@ -137,6 +139,13 @@ func NewService() (svr *Service, err error) {
 		log.Info("frps kcp listen on udp %s:%d", cfg.BindAddr, cfg.KcpBindPort)
 	}
 
+	// Listen for accepting connections from client using websocket protocol.
+	websocketPrefix := []byte("GET /%23frp")
```

Comment:
We can not judge the protocol is websocket just by the prefix. I test when the bind_port equal the v_http_port,  it transports failed

Reply:
`/#frp` or other uniq string is a special path prefix of frp. It's a simple and efficient way to detect protocol used for frp in case reading too many letters or facing other problems.

We don't need to care aboud the coincidence if others used the same path which i think it has low probility. In other words, we could let the prefix more unique and short, maybe using some special letters.

Reply:
OK! I understand!

Reply:
Do you have any idea on the sepcial path? Such as `(~!)f` ?

Reply:
I think "#frp" is already unique if not test for it